modules/git/git.go

```diff
@@ -161,68 +180,72 @@ func Init(ctx context.Context) error {
 		globalCommandArgs = append(globalCommandArgs, "-c", "uploadpack.allowfilter=true", "-c", "uploadpack.allowAnySHA1InWant=true")
 	}
 
-	// Save current git version on init to gitVersion otherwise it would require an RWMutex
-	if err := LoadGitVersion(); err != nil {
-		return err
-	}
-
 	// Git requires setting user.name and user.email in order to commit changes - if they're not set just add some defaults
 	for configKey, defaultValue := range map[string]string{"user.name": "Gitea", "user.email": "gitea@fake.local"} {
-		if err := checkAndSetConfig(configKey, defaultValue, false); err != nil {
+		if err := configSet(configKey, defaultValue); err != nil {
```

Comment:
`false` means don't force set the value. It's still necessary.

Reply:
ps: why it's not needed:
* before: this `force` argument is used for `username`/`email` only, to avoid changing user's gitconfig
* now: no conflict anymore, Gitea can always set `username`/`email`  safely.